cms/core/wagtail_hooks.py

```diff
@@ -35,3 +43,16 @@ def global_admin_css() -> str:
 
 register_snippet(ContactDetailsViewSet)
 register_snippet(GlossaryViewSet)
+
+
+@hooks.register("after_edit_page")
+def after_edit_page(request: "HttpRequest", page: "Page") -> None:
+    if page.locale.language_code != settings.LANGUAGE_CODE:
+        return
+
+    # Check if proper translations of the page exist, which are not simple aliases
+    has_proper_translations = page.get_translations().filter(alias_of__isnull=True).exists()
+    if has_proper_translations:
+        messages.warning(
+            request, "A translated version of this page exists. If you make any changes, please make sure to update it."
+        )
```

Comment:
Non-blocker: Bonus point if we can link to it.

Reply:
Ah, good point. I suspect there is no link to view "all translations" for a current page link?

Reply:
There is not, but let's go with the first translation for now.
And worry about 2+ later

Reply:
Thanks both, addressed in [f6d8364](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/f6d83642938e5d41ad31d69b6577d83e41f06697)